src/libs/actions/User.js

```diff
@@ -65,6 +65,56 @@ function getBetas() {
     });
 }
 
+/**
+ * Fetch the public domain info for the current user (includes secondary logins).
+ *
+ * This API is a bit weird in that it sometimes depends on information being cached in bedrock.
+ * If the info for the domain is not in bedrock, then it creates an asynchronous bedrock job to gather domain info.
+ * If that happens, this function will automatically retry itself in 10 minutes.
+ *
+ * @param {String} loginList
+ */
+function getDomainInfo(loginList) {
```

Comment:
I am probably just not seeing why it's necessary but we can access the `loginList` from Web-Expensify so I'm confused about why we pass it as a parameter here?

Reply:
The idea is that we would filter the `loginList` on the frontend and only make API calls with logins that are not in the common domain list. So for example, if the `loginList` was `['a@gmail.com', 'b@yahoo.com', 'c@privatedomain.com']` we would only call the API to check `c@privatedomain.com`. If we were to access the `loginList` in Web-E, we would have to filter the list again.